frontend/src/components/MeetingRoom/AudioInputOutputDevices/AudioInputOutputDevice.spec.tsx

```diff
@@ -61,12 +100,22 @@ describe('AudioInputOutputDevice Component', () => {
     );
 
     const outputDevicesElement = screen.getByTestId('output-devices');
-    expect(outputDevicesElement).toBeInTheDocument();
+    await waitFor(() => expect(outputDevicesElement.children).to.have.length(3));
+    const firstChild = outputDevicesElement.firstChild as HTMLOptionElement;
```

Comment:
nit: can `firstChild` be renamed to something more self-explanatory? something like `firstDeviceElement`?

Reply:
[firstChild](https://developer.mozilla.org/en-US/docs/Web/API/Node/firstChild) is an inherited property in the DOM.

Reply:
oh, thanks for the info! we can make `const`s be whatever we'd like them to be, though 🙂 it's just a nit though so if Mike prefers to keep it as a `firstChild` I'm okay with that